source/common/access_log/access_log_impl.cc

```diff
@@ -118,7 +118,7 @@ bool RuntimeFilter::evaluate(const RequestInfo::RequestInfo&,
     random_value = random_.random();
   }
 
-  return runtime_.snapshot().featureEnabled(
+  return runtime_.snapshot().sampleFeatureEnabled(
```

Comment:
For the moment, undo all changes to featureEnabled(). Lets not support runtimes for the fractional percent. You can simply do a if percent != 0 (old path) else (new path)